cardano-cli/src/Cardano/CLI/EraBased/Transaction/Command.hs

```diff
@@ -243,11 +246,22 @@ data TransactionCalculateMinValueCmdArgs era = TransactionCalculateMinValueCmdAr
   deriving Show
 
 data TransactionCalculatePlutusScriptCostCmdArgs = TransactionCalculatePlutusScriptCostCmdArgs
-  { nodeConnInfo :: !LocalNodeConnectInfo
+  { nodeContextInfo :: !NodeContextInfo
   , txFileIn :: FilePath
   , outputFile :: !(Maybe (File () Out))
   }
 
+data NodeContextInfo
```

Comment:
These types are  missing haddocks explaining what this type is.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/1079/commits/c722a04785977e6437d69fa41544c7abb139bf43

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/1079/commits/0e4186f2727a1bb67feafcb3245f1ab05cc2c5cc